pkg/controller/performanceprofile/components/kubeletconfig/kubeletconfig.go

```diff
@@ -116,6 +117,17 @@ func New(profile *performancev2.PerformanceProfile, profileMCPLabels map[string]
 						},
 					}
 				}
+
+				// require full physical CPUs only to ensure maximum isolation
+				if topologyPolicy == kubeletconfigv1beta1.SingleNumaNodeTopologyManagerPolicy {
```

Comment:
We need to make sure this works with the kubeletconfig.experimental annotation.

Reply:
I don't recall this annotation, could you please add a link?

Reply:
it was: https://github.com/openshift-kni/performance-addon-operators/blob/master/docs/kubelet-config-snippets.md

Reply:
so, we can force the option off if needed using the annotation:
https://github.com/openshift-kni/performance-addon-operators/pull/790/commits/c91534895179be901561bf4e49a105139991978e#diff-0f0074e84690443303677c9a59e7c67616cb32d197bdec89eeb9bae667b05d83R153